agent-lib/src/main/java/org/terracotta/angela/agent/kit/LocalKitManager.java

```diff
@@ -45,6 +47,7 @@
   private static final Logger logger = LoggerFactory.getLogger(LocalKitManager.class);
   private final Map<String, File> clientJars = new HashMap<>();
   private final KitResolver kitResolver;
+  public static final String ANGELA_LOCK_FILE = "angela-install.lock";
```

Comment:
Why does it need to be public?

Also, renaming it to something like `INSTALLATION_LOCK_FILE_NAME` would help. (`ANGELA_LOCK_FILE` is too generic)